subprojects/plugins/src/integTest/groovy/org/gradle/api/tasks/bundling/JarIntegrationTest.groovy

```diff
@@ -26,6 +27,7 @@ import java.util.jar.Manifest
 
 class JarIntegrationTest extends AbstractIntegrationSpec {
 
+    @TestReproducibleArchives
```

Comment:
does it make sense to just split out the tests which shouldn't have `@TestReproducibleArchives`?